src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,15 +17,22 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
```

Comment:
see https://github.com/ceph/ceph/pull/14739/commits/0598d533a150115f7580c2cc63124ef90c2e433c#r112856481 in #14739

i am pasting my comments here also

> i'd suggest always use plain `diff` without `-y` option or just `cmp` to compare `checkcsjson` and `csjson`. and could also wrap this logic with a simple helper function, as it is repeating itself multiple times in this shell script.
> 
> @dzafman what do you think?